Documentation/BasicPrinciples.rst

```diff
@@ -135,8 +135,6 @@ A guide is a manual.
 
 Guides offer advice on **how best to achieve a given task**. They are goal oriented.
 
-:ref:`Read more ... <writing-guide>`
```

Comment:
Wouldnt the link be good?" to keep?

Reply:
The referenced content was removed in a previous  PR